src/engine/config_toolset.bat

```diff
@@ -165,21 +166,25 @@ if "_%BOOST_JAM_TOOLSET_ROOT%_" == "__" (
     if NOT "_%VS150COMNTOOLS%_" == "__" (
         set "BOOST_JAM_TOOLSET_ROOT=%VS150COMNTOOLS%..\..\VC\"
     ))
-SET cl_path_cmd="%~dp0..\tools\vc141helper\cl_path.cmd"
+
+IF DEFINED EXPENSIVE_POWERSHELL2 goto :Skip_VC1410
+SET EXPENSIVE_POWERSHELL2=1
+SET vsCOMhelper_cmd="%~dp0..\tools\vsCOMhelper\get_key_helper.cmd" "IsVcCompatible,VisualCppToolsVersionMinor=14.10" "InstallationPath"
```

Comment:
@KindDragon I took the filtering idea from you, it's a very good idea 👍 I'm using it in all other projects...